src/components/CreateShoppingListForm/CreateShoppingListForm.js

```diff
@@ -2,61 +2,96 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 
+import StoreSearch from '../StoreSearch';
+
 import './CreateShoppingListForm.css';
 import { createList } from '../../actions/shoppingLists';
+import { clearCurrentStore } from '../../actions/yelpAPI';
 
 export class CreateShoppingListForm extends React.Component {
-  onSubmit(event) {
-    const { dispatch, history } = this.props;
+  constructor(props) {
+    super(props);
+    this.state = { wantStore: false };
+  }
```

Comment:
This is a WIP feature, getting rid of the StoreSearch component if the user clicks a button that they don't want a store associated with their shopping list